backend/equipmentposts/models.py

```diff
@@ -8,4 +11,18 @@ class EquipmentPost(Post):
     equipment_type = models.TextField()
 
     class Meta:
-        app_label = 'equipmentposts'
\ No newline at end of file
+        app_label = 'equipmentposts'
+
+
+class Comment(models.Model):
+    parent_equipment = models.ForeignKey(EquipmentPost, on_delete=models.CASCADE)
+    content = models.TextField()
+    owner = models.ForeignKey(User, on_delete=models.CASCADE, related_name="equipment_comment")
```

Comment:
The default_names in Comment and Answer models are defined but unused, is that correct?

Reply:
If you are talking about the related_name field, yes we are not using it explicitly but Django needs them to distinguish between equipment comments/answers and event comments/answers, because both models have the same name. We need to specify a related_name field to help it differentiate between the two.

Reply:
Got it, thank you!